.github/actions/spelling/action.yml

```diff
@@ -168,26 +168,27 @@ runs:
           cspell:companies/src/companies.txt
           cspell:cryptocurrencies/cryptocurrencies.txt
           cspell:csharp/csharp.txt
-          cspell:css/css.txt
+          cspell:css/dict/css.txt
           cspell:docker/src/docker-words.txt
-          cspell:dotnet/dotnet.txt
+          cspell:dotnet/dict/dotnet.txt
           cspell:filetypes/filetypes.txt
           cspell:fonts/fonts.txt
-          cspell:fullstack/fullstack.txt
-          cspell:html/html.txt
+          cspell:fullstack/dict/fullstack.txt
+          cspell:html-symbol-entities/entities.txt
+          cspell:html/dict/html.txt
+          cspell:html/src/svg.txt
           cspell:lorem-ipsum/dictionary.txt
           cspell:node/additional_words.txt
-          cspell:node/node.txt
-          cspell:npm/npm.txt
-          cspell:powershell/powershell.txt
+          cspell:node/dict/node.txt
+          cspell:npm/dict/npm.txt
+          cspell:powershell/dict/powershell.txt
           cspell:public-licenses/src/generated/public-licenses.txt
-          cspell:software-terms/src/network-os.txt
-          cspell:software-terms/src/network-protocols.txt
-          cspell:software-terms/src/software-terms.txt
```

Comment:
@BenedekFarkas we had this both here and under `default-dictionaries` above. Can you tell why this is the case? Actually, why do we need to use `check_extra_dictionaries` at all, couldn't we put everything into `default-dictionaries` (since these here in `check_extra_dictionaries` are also default dictionaries, but ones that can't be overridden)?

Reply:
Those have a very different purpose:

- `default-dictionaries` are those that we selected as general-purpose enough so that probably every or almost every consumer project will need them. It can be overridden, but if you just want to add to it, you can use `additional-dictionaries`. Those two will be merged in the `Merge dictionary lists` step, producing `merged-dictionary-list`, which will be passed on to the spelling action as `extra_dictionaries` (see: https://github.com/check-spelling/check-spelling/blob/main/action.yml#L68).
Dictionaries that are used for the actual spell checking should be kept to a minimum to avoid false negatives.
- `check_extra_dictionaries` will be passed on directly (see https://github.com/check-spelling/check-spelling/blob/main/action.yml#L72), but this list of dictionaries will be used to suggest dictionaries when the spellchecking report comes up with unrecognized entries. The entries that are also found `default-dictionaries` are kept here, so even if you override `default-dictionaries` (and remove something), the spell checking report can still suggest such a useful dictionary to be re-added.

Reply:
So `check_extra_dictionaries` is solely for this?

![image](https://github.com/Lombiq/GitHub-Actions/assets/1976647/3181888c-7da2-4410-b6d3-f7ecfde153f7)

Since `check-spelling` is suggesting dictionaries from https://github.com/check-spelling/cspell-dicts/tree/main/dictionaries (or [the defaults](https://github.com/check-spelling/check-spelling/blob/ed14e107b2404ceec4db32161bc3cc353b123b46/action.yml#L75) too (that's how I fixed broken dictionary paths), I don't think we need to include `cspell` ones there.

Reply:
This whole section is influenced by that:
![image](https://github.com/Lombiq/GitHub-Actions/assets/2797449/d38af990-f6c6-4f09-b0e5-0a3fbb1241bc)

The reason why any dictionary shows up in the first place is due to [the default value in check-spelling](https://github.com/check-spelling/check-spelling/blob/ed14e107b2404ceec4db32161bc3cc353b123b46/action.yml#L72), but [the override](https://github.com/Lombiq/GitHub-Actions/blob/e092b644eb21256f4f4057bbc92142201657a615/.github/actions/spelling/action.yml#L166) trims it down to entries that are likely to be relevant to us (and added our own).

But [a comment similar to the one about upgrading to a new version](https://github.com/Lombiq/GitHub-Actions/blob/e092b644eb21256f4f4057bbc92142201657a615/.github/actions/spelling/action.yml#L152-L154) would've probably helped (to check if our override list is still working OK), as it seems that the existing dictionary paths too can break between releases.

Reply:
Got it, thanks.